### PR TITLE
[docs] Add a note that cdpPort is required when you override the user dir

### LINKED ISSUE #5627: Video recording is not supported by chrome browser when using --user-data-dir
testcafe version 1.9.1
useragent: "Chrome 85.0.4183.121 / Linux 0.0"

1. Add flag --user-data-dir to chrome browser
2. Result: Videos will not be recorded
3. Expected: Videos to record even when using a specific --user-data-dir since it is convenient to define where the browser will store all navigation related data
4. Remove --user-data-dir from chrome browser
5. Videos will be recorded

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
fixes #5627

### CODE REVIEW

**Review by AlexKamaev — APPROVED** [review]

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Comment by VasilyStrelyaev** [comment]
@AndreyBelym 

That's what the response (`400`) has in the `body`:

image

**Comment by VasilyStrelyaev** [comment]
Do we need to use a different user agent?

File: docs/articles/documentation/guides/concepts/browsers.md [hunk 1]
```diff
@@ -334,4 +334,10 @@ When you pass the `:userProfile` flag to a portable browser, also use the [brows
 testcafe chrome:d:\chrome_portable\chrome.exe:userProfile tests/test.js
 ```
 
-> It is not recommended that you combine the :userProfile flag with either headless or emulation mode, because this can lead to unstable behavior.
+Set the [cdpPort](#emulator-parameters) property to `9222` if you use the `:userProfile` flag and video recording in Chrome and Chromium-based browsers:
```
Comment (DIRECTcut) [inline docs/articles/documentation/guides/concepts/browsers.md:337]: Not sure I understand this sentence. Do you mean:

```suggestion
If you use the `:userProfile` flag, set the [cdpPort](#emulator-parameters) property to `9222`  to record videos in Chrome and Chromium-based browsers.
```

**Review by DIRECTcut — APPROVED** [review]

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**